### DISCUSSION TITLE
Tmux statusline theme support?

### DISCUSSION THREAD

**Discussion by NovaViper** [nix-community/stylix#296] [Q&A] [answered]
Hey I just came across this project and have started integrating it into my flake but I'm wondering if it's possible to use the colors it generates to color any of the statusline themes for Tmux? Originally i used [dracula-tmux](https://github.com/dracula/tmux) and I'm wondering if I could still use the statusline it makes but have the colors made from stylix to be used for it instead.

**Comment by danth** [comment]
We do have a module for Tmux which should set itself up automatically, if you have `programs.tmux.enable` in your Home Manager config.

I'm not familiar with Tmux, so I'm not sure if it's possible to install the Dracula layout without the colors.

**Reply by NovaViper** [reply]
Hey sorry for the delay! I tried the module and found out that the tmux statusline themes (specifically the ones that aren't the default one like [tmux-nova](https://github.com/o0th/tmux-nova) or [tmux-powerlines](https://github.com/erikw/tmux-powerline)) in fact DO NOT take from the colorscheme that is set for Tmux.

🤔 Is there a way to refer to the color scheme directly from stylix? Something like `config.lib.stylux.baseColorCode`?

**Reply by danth** [reply, marked as answer]
Yes, you can use `config.lib.stylix.colors.baseXX` or `config.lib.stylix.colors.withHashtag.baseXX`

**Reply by NovaViper** [reply]
Ah cool, thank you!

**Reply by eveeifyeve** [reply]
> Yes, you can use `config.lib.stylix.colors.baseXX` or `config.lib.stylix.colors.withHashtag.baseXX`

It would be nice to take consideration for these in stylix.
